volto_addon/{{ cookiecutter.addon_name }}/README.md

```diff
@@ -45,19 +49,47 @@ Start volto with:
 yarn start
 ```
 
+### Into a cookiecutter-volto based project
+
+This is the way to install it in the (upcoming) Volto 18.
+
+Add `{{ cookiecutter.npm_package_name }}` to your `package.json`:
+
+```JSON
+"dependencies": {
+    "{{ cookiecutter.npm_package_name }}": "*"
+}
+```
+
+Add `{{ cookiecutter.npm_package_name }}` to your `volto.config.js`:
+
+```javascript
+const addons = ['{{ cookiecutter.npm_package_name }}'];
+```
+
+If this package provides a volto theme, and you want to activate it (and only then!), add to your `volto.config.js`:
+
+```javascript
+const theme = '{{ cookiecutter.npm_package_name }}';
+```
+
 ### Test it
 
 Go to http://localhost:3000/, login and check the awesome new features.
 
+
 ## Development
 
 The development of this add-on is done in isolation using a new approach using pnpm workspaces and latest `mrs-developer` and other Volto core improvements.
 For this reason, it only works with pnpm and Volto 18 (currently in alpha)
 
-### Requisites
+### Preprequisites
```

Comment:
Wait, are these pre-requisites required only for development or for creating a project as well? If the latter, then this section must be moved immediately before Installation.

```suggestion
### Pre-requisites
```

Reply:
This are the per-requisites for development.